# My Jupyter Notebook on IBM Watson Studio

**Millicent  Patrick**
                          
  Data Scientist                        
  

*I am interested in Data Science because am passionate in making positive change by gaining insights from data.*

### The coding below illustrates Basic Mathematical operations using Python language

In [2]:
5+4

9

In [3]:
 5-4

1

In [4]:
5*4

20

In [5]:
6/3

2.0

In [6]:
5**4

625

In [7]:
6//3

2




`ADVANCED FEATURES USING MARKDOWN IN JUPYTER NOTEBOOK EXAMPLES`
----

* Bulletted list
* Numbered List
* Tables
* Rendered Image
* Highlighting
* Horizontal line
* Hyperlinks
* Strikethrough

----       

   **Numbered List of mathematical operations in the code above**

   1. Addition
   2. Subtraction
   3. Multiplication
   4. Division
   5. Exponential
   6. Quotient

----
   
                               TABLE OF OPERATIONS AND ANSWERS OF THE CODING ABOVE
   
   |  Operation       |  Calculation  |  Answer  |
   |:-----------      |   :---------: |  -------:|
   |1.  Addition      | 5 + 4         | 9        |
   |__________________|_______________|__________|
   |2.  Subtraction   | 5 - 4         | 1        |
   |__________________|_______________|__________|
   |3.  Multiplication| 5 * 4         | 20       |
   |__________________|_______________|__________|
   |4.  Division      | 6 / 3         | 2        |
   |__________________|_______________|__________|
   |5.  Exponential   | 5 ** 4        | 625      |
   |__________________|_______________|__________|
   |6.  Quotient      | 6 // 3        | 2        |
   |__________________|_______________|__________|
   
   


   
   
    
   
          